docs/topics/data-analysis/lets-plot.md

```diff
@@ -0,0 +1,200 @@
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
+2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+
+   > Alternatively, access the Kotlin Notebook plugin from **Settings** | **Plugins** | **Marketplace** within IntelliJ IDEA.
+   >
+   {type="tip"}
+
+3. Create a new notebook by selecting **File** | **New** | **Kotlin Notebook**.
+4. In your notebook, import the LPK and Kotlin DataFrame libraries by running the following command:
+
+    ```kotlin
+    %use lets-plot
+    %use dataframe
+    ```
+
+## Prepare the data
+
+Create a DataFrame that stores simulated numbers of the monthly average temperature in three cities: Berlin, Madrid, and Caracas.
+
+Use the `dataFrameOf()` function from the Kotlin DataFrame library to generate the DataFrame. Paste and run the following code snippet in your Kotlin Notebook:
+
+```kotlin
+// The months variable stores a list with the 12 months of the year
+val months = listOf(
+    "January", "February",
+    "March", "April", "May",
+    "June", "July", "August",
+    "September", "October", "November",
+    "December"
+)
+// The tempBerlin, tempMadrid, and tempCaracas variables store a list with temperature values for each month
+val tempBerlin =
+    listOf(-0.5, 0.0, 4.8, 9.0, 14.3, 17.5, 19.2, 18.9, 14.5, 9.7, 4.7, 1.0)
+val tempMadrid =
+    listOf(6.3, 7.9, 11.2, 12.9, 16.7, 21.1, 24.7, 24.2, 20.3, 15.4, 9.9, 6.6)
+val tempCaracas =
+    listOf(27.5, 28.9, 29.6, 30.9, 31.7, 35.1, 33.8, 32.2, 31.3, 29.4, 28.9, 27.6)
+
+// The df variable stores a DataFrame of three columns, including records of months, temperature, and cities
+val df = dataFrameOf(
+    "Month" to months + months + months,
+    "Temperature" to tempBerlin + tempMadrid + tempCaracas,
+    "City" to List(12) { "Berlin" } + List(12) { "Madrid" } + List(12) { "Caracas" }
+)
+df.head(4)
+```
+
+You can see that the DataFrame has three columns: Month, Temperature, and City. The first four rows of the DataFrame
+contain records of the temperature in Berlin from January to April:
+
+![Dataframe exploration](visualization-dataframe-temperature.png){width=600}
+
+To create a plot using the LPK library, you need to convert your data (`df`) into a `Map` type that stores the
+data in key-value pairs. You can easily convert a DataFrame into a `Map` using the `.toMap()` function:
+
+```kotlin
+val data = df.toMap()
+```
+
+## Create a scatter plot
+
+Let's create a scatter plot in Kotlin Notebook with the LPK library. 
+
+Once you have your data in the `Map` format, use the `letsPlot()` function from the LPK library to generate the scatter plot. 
+You can specify the values for the X and Y axes, as well as define categories with colors. Additionally, 
```

Comment:
```suggestion
You can specify the values for the X and Y axes, as well as define color categories. Additionally, 
```

Reply:
I'll leave it as "categories and their color" because we need to emphasize categories generation (being these categorical values for the X and Y access), plus their colors.